app/src/main/java/org/wikipedia/feed/configure/ConfigureViewModel.kt

```diff
@@ -0,0 +1,64 @@
+package org.wikipedia.feed.configure
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.CoroutineExceptionHandler
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+import org.wikipedia.dataclient.ServiceFactory
+import org.wikipedia.dataclient.WikiSite
+import org.wikipedia.feed.FeedContentType
+import org.wikipedia.util.Resource
+
+class ConfigureViewModel() : ViewModel() {
+
+    private val handler = CoroutineExceptionHandler { _, throwable ->
+        _uiState.value = Resource.Error(throwable)
+    }
+
+    private val _uiState = MutableStateFlow(Resource<Boolean>())
+    val uiState = _uiState.asStateFlow()
+
+    init {
+        loadFeedAvailability()
+    }
+
+    private fun loadFeedAvailability() {
+        _uiState.value = Resource.Loading()
+        viewModelScope.launch(handler) {
+            val result = ServiceFactory.getRest(WikiSite("wikimedia.org")).feedAvailability()
```

Comment:
Let's no longer use a hardcoded `"wikimedia.org"` here. This API call can be made to any wikimedia domain, so we can just use `WikipediaApp.instance.wikiSite`.

Reply:
Sure! Done.